kernel/userspace.c

```diff
@@ -221,7 +225,7 @@ void *_impl_k_object_alloc(enum k_objects otype)
 
 	dyn_obj = z_thread_malloc(sizeof(*dyn_obj) + obj_size_get(otype));
 	if (!dyn_obj) {
-		SYS_LOG_WRN("could not allocate kernel object");
+		LOG_WRN("could not allocate kernel object");
```

Comment:
This the only place we use logging in the kernel??

Reply:
right now yes, I think we will have more once this is moved to the new logger.